server/cli/src/backup/backup.rs

```diff
@@ -0,0 +1,139 @@
+use super::*;
+use chrono::Utc;
+use copy_dir::copy_dir;
+use service::settings::Settings;
+use std::{fs, io, path::PathBuf, process::Command, str::FromStr};
+
+pub(crate) fn backup(settings: &Settings) -> Result<(), BackupError> {
+    let DirSettings {
+        backup_dir,
+        pg_bin_dir,
+    } = get_dirs_from_settings(settings)?;
+
+    let Dirs {
+        backup_name,
+        file_dir,
+        database_dir,
+    } = create_backup_dir(backup_dir)?;
+
+    copy_files(settings, &file_dir)?;
+
+    // Backup database
+    if cfg!(feature = "postgres") {
+        dump_postgres_database(settings, &database_dir, pg_bin_dir)?;
+    } else {
+        copy_sqlite_files(settings, &database_dir)?;
+    }
+
+    println!("{backup_name}");
```

Comment:
when using interactively a 'backup completed' would be nice
```suggestion
    println!("Backup completed in folder {backup_name}");
```

Reply:
It looks like support will not be using the stdio, happy with this ! Thanks